docs/contributing/Contributor-access.md

```diff
@@ -51,7 +51,19 @@ Being in the Discord "team" group gives you the following:
 
 ## Process
 
-The actual process (as followed by Flutter repo admins) is as follows:
+### Requesting access
```

Comment:
So I winder if this is an update we need to make to the Google onboarding docs. We don't encourage folks to ask to be added. Having write access to Flutter repos, while necessary for Googlers to do some of their work, is not necessary for contributors. It's a trusted status that folks are sponsored for.

Maybe that is the update, with a history of regular contribution, a team member may nominate you?

I get requests semi regularly, which is a bummer to say no to, but write access is really only appropriate for trusted and active contributors.